index.html

```diff
@@ -19,14 +19,123 @@ <h1 class="main-text">I'M ZALIHA GLAD TO SEE YOU</h1>
 				<p>I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
 			</section>
 			<a href="#" class="connect">LET'S CONNECT</a>
-					<div class="icons">
-							<img src="./images/twitter.png" alt="">
-							<img src="./images/Linkedin.png" alt="">
-							<img src="./images/vector-m.png" alt="">
-							<img src="./images/baby.svg" alt="">
-							<img src="./images/hand.png" alt="">
-							
+				<div class="icons">
+						<img src="./images/twitter.png" alt="">
+						<img src="./images/Linkedin.png" alt="">
+						<img src="./images/vector-m.png" alt="">
+						<img src="./images/baby.svg" alt="">
+						<img src="./images/hand.png" alt="">
+						
+				</div>
```

Comment:
Add all of your images for your social media inside the `a` tag (links). For instance:
```
<a href="#">
  <img src="image_path" alt="image_name">
</a>
```

- [ ] Add your social media icons inside link tags.